transcribe.py

```diff
@@ -131,9 +132,12 @@ def main(args, ext_json = ['.json', '.json.gz']):
 		try:
 			tic = time.time()
 			y, ylen = y.to(args.device), ylen.to(args.device)
+			print('Input:', audio_name)
```

Comment:
Please add full audio_path as well, it's more useful for debugging

Reply:
done

Reply:
maybe the fix is incoming, can't see it yet